### DISCUSSION TITLE
Setting up g-helper for horn strix g16

### DISCUSSION THREAD

**Discussion by eternalEVG** [seerge/g-helper#3477] [General]
What settings should I use to make my laptop work better?

What values ​​should I set for PL1 and PL2 in the “Silent”, “Balanced” and “Turbo” modes?

Do I need to enable CPU boost? 

And also, what values ​​should I set in GPU mode in the same three modes?

My specifications: asus rog strix g16 i5 13450hx and 4060, 16gb ram, 1tr ssd

![Снимок экрана 2024-12-12 221159](https://github.com/user-attachments/assets/2e426b38-1cab-4450-ab9d-d82ed2b476db)
![Снимок экрана 2024-12-12 221217](https://github.com/user-attachments/assets/1026d89f-e854-4123-9eb5-9843110e966f)

**Comment by seerge** [comment]
@eternalEVG hello, each mode is baked in BIOS and comes with some  power limits tuned for your model by manufacturer. So unless you pursue some specific goals - there is no need to apply custom limits (or fan curves).

**Comment by spolaorantonello-blip** [comment]
rog strix g16jmr 2025 con rtx 5060  come impostare illuminazione a 4 zone personalizzata.Ho scoperto che ha la tastiera per key ma asus l ha castrata
